src/layout/index.jsx

```diff
@@ -9,15 +9,23 @@ import "bootstrap/dist/css/bootstrap.min.css";
 import "./css/style.scss";
 import favicon from "../../static/logos/logo.png";
 
+
 // eslint-disable-next-line no-unused-vars
 export default ({ children, location }) => (
-  <body>
+  <React.Fragment>
     <Helmet>
       <meta name="description" content={config.siteDescription} />
       <link rel="icon shortcut" href={favicon} type="image/png" />
+      <meta property="og:title" content={config.siteTitle} />
```

Comment:
Using siteTitle property won't be a good choice here incase someone share a link to a some different page, that would only show the website title. If someone shares the link to about page it should display something like `About - Terasology`

You can have a look [here](https://neilpatel.com/wp-content/uploads/2014/03/Facebook-2014-03-03.jpg).

Reply:
What do you think of using siteTitleShort instead of siteTitle?